meadow-client/src/Types.purs

```diff
@@ -115,154 +64,117 @@ type ChildSlot
 
 data MarloweError
   = MarloweError String
 
-_view ::
-  forall s a.
-  Lens' {view :: a | s} a
+_view :: forall s a. Lens' {view :: a | s} a
 _view = prop (SProxy :: SProxy "view")
 
-_runResult ::
-  forall s a.
-  Lens' {runResult :: a | s} a
+_runResult :: forall s a. Lens' {runResult :: a | s} a
 _runResult = prop (SProxy :: SProxy "runResult")
 
-_authStatus ::
-  forall s a.
-  Lens' {authStatus :: a | s} a
+_marloweCompileResult :: forall s a. Lens' {marloweCompileResult :: a | s} a
+_marloweCompileResult = prop (SProxy :: SProxy "marloweCompileResult")
+
+_authStatus :: forall s a. Lens' {authStatus :: a | s} a
 _authStatus = prop (SProxy :: SProxy "authStatus")
 
-_createGistResult ::
-  forall s a.
-  Lens' {createGistResult :: a | s} a
+_createGistResult :: forall s a. Lens' {createGistResult :: a | s} a
 _createGistResult = prop (SProxy :: SProxy "createGistResult")
 
-_marloweState ::
-  forall s a.
-  Lens' {marloweState :: a | s} a
+_marloweState :: forall s a. Lens' {marloweState :: a | s} a
 _marloweState = prop (SProxy :: SProxy "marloweState")
 
-
 -- Oracles should not be grouped (only one line per oracle) like:
 --    Oracle 3: Provide value [$value] for block [$timestamp]
-type OracleEntry = { timestamp :: BlockNumber -- editable
-                   , value :: BigInt }        -- editable
+type OracleEntry
+  = {timestamp :: BlockNumber, value :: BigInt}
 
+-- editable
 _timestamp ::
   forall s a.
   Lens' {timestamp :: a | s} a
 _timestamp = prop (SProxy :: SProxy "timestamp")
 
-_value ::
-  forall s a.
-  Lens' {value :: a | s} a
+_value :: forall s a. Lens' {value :: a | s} a
 _value = prop (SProxy :: SProxy "value")
 
-type InputData = { oracleData :: Map IdOracle OracleEntry -- oracle inputs (before person
-                 , inputs :: Map Person PersonInput } -- inputs (grouped by Person id)
+type InputData
+  = {oracleData :: Map IdOracle OracleEntry, inputs :: Map Person PersonInput}
 
+-- inputs (grouped by Person id)
 _oracleData ::
   forall s a.
   Lens' {oracleData :: a | s} a
 _oracleData = prop (SProxy :: SProxy "oracleData")
 
-_inputs ::
-  forall s a.
-  Lens' {inputs :: a | s} a
+_inputs :: forall s a. Lens' {inputs :: a | s} a
 _inputs = prop (SProxy :: SProxy "inputs")
 
-type TransactionData = { inputs :: Array (Either (Tuple Person PersonInput) (Tuple IdOracle OracleEntry))
-                                  -- ^ not grouped (but participants and oracles labelled inline)
-                                  -- e.g: Participant 1 - Action $IdAction - Commit ....
-                                  --      Oracle 3 - Provide value $value for block $timestamp
-                       , signatures :: Map Person Boolean -- checkboxes
-                       , outcomes :: Map Person BigInt } -- table under checkboxes
+type TransactionData
+  = {inputs :: Array (Either (Tuple Person PersonInput) (Tuple IdOracle OracleEntry)), signatures :: Map Person Boolean, outcomes :: Map Person BigInt}
```

Comment:
You've lost the comment here. Is that deliberate, or a `purty` bug?

Reply:
ugh, purty

Reply:
:-(